apps/challenges/aws_utils.py

```diff
@@ -1195,3 +1200,20 @@ def challenge_approval_callback(sender, instance, field_name, **kwargs):
                             challenge.id, failures[0]["message"]
                         )
                     )
+
+
+signals.post_save.connect(
```

Comment:
@burnerlee we can't move models post save callback to `aws_utils`. As the name of the file suggests this file is only for code related to aws services util methods. Please move `post_save` callbacks to `challenges.models.py`